index.html

```diff
@@ -353,34 +353,35 @@ <h2 class="lastpage-heading">I'm always interested in hearing about new projects
         <ul class="icon-list last-page-footer">
             <li class="img-list">
                 <a  href="https://github.com/Mehria-Rahmani/Hello-Microverse.git" target="_blank">
-                    <img class="github icons" alt="Github Icon" src="assets/icon/Disabled(2).png">
+                    <img class="github icons" alt="Github Icon" src="assets/github.svg">
                 </a>
             </li>
             <li class="img-list">
                 <a href="https://www.linkedin.com/in/mehria-rahmani-379211222" target="_blank">
-                    <img class="linkedin icons" alt="LinkedIn Icon" src="assets/icon/Frame.png">
+                    <img class="linkedin icons" alt="LinkedIn Icon" src="assets/Linkedin.svg">
```

Comment:
- [ ] [OPTIONAL] Same issue here, the LinkedIn icon name is `linkedin.svg`

Reply:
- [ OPTIONAL] just fix this Issue as previous review suggested, and consider your icon colors as well, they should be the same one from Figma template
![image](https://user-images.githubusercontent.com/43877689/173193916-d3525c7a-497c-4281-9240-e823e8b8c22f.png)